libvmi/driver/kvm/kvm.c

```diff
@@ -132,11 +142,16 @@ exec_xp(
     int numwords,
     addr_t paddr)
 {
-    char *query = (char *) safe_malloc(256);
+    char *query = (char *) safe_malloc(QMP_CMD_LENGTH);
 
-    sprintf(query,
-            "'{\"execute\": \"human-monitor-command\", \"arguments\": {\"command-line\": \"xp /%dwx 0x%x\"}}'",
+    int rc = snprintf(query,
```

Comment:
Not having a blank line turns these into awful walls of text and looks more inconsistent to me because of things like the way `char *output` is defined below.